inst/assets/BS4/pkgdown.js

```diff
@@ -64,5 +64,85 @@
 
     });
   }
+
+    /* Search marking --------------------------*/
+    var url = new URL(window.location.href);
+    var toMark = url.searchParams.get("q");
+    var mark = new Mark("div.col-md-9");
+    if (toMark) {
+      mark.mark(toMark, {
+        accuracy: {
+          value: "complementary",
+          limiters: [",", ".", ":", "/"],
+        }
+      });
+    }
+
+  /* Search --------------------------*/
+    // Initialise search index on focus
+  var fuse;
+  $("#search-input").focus(async function(e) {
+    if (fuse) {
+      return;
+    }
+
+    $(e.target).addClass("loading");
+    var response = await fetch($("#search-input").data("search-index"));
```

Comment:
in a production website the search-index is at /search.json but in a dev website it is at /dev/search.json.